site/src/theme/colors.ts

```diff
@@ -4,43 +4,43 @@ export const colors = {
   white: "hsl(0, 0%, 100%)",
 
   gray: {
-    17: "hsl(220, 50%, 3%)",
-    16: "hsl(223, 44%, 9%)",
-    15: "hsl(222, 38%, 14%)",
-    14: "hsl(222, 32%, 19%)",
-    13: "hsl(222, 31%, 25%)",
-    12: "hsl(222, 30%, 31%)",
-    11: "hsl(219, 29%, 38%)",
-    10: "hsl(219, 28%, 45%)",
-    9: "hsl(219, 28%, 52%)",
-    8: "hsl(218, 29%, 58%)",
-    7: "hsl(219, 30%, 64%)",
-    6: "hsl(219, 31%, 71%)",
-    5: "hsl(218, 32%, 77%)",
-    4: "hsl(223, 38%, 84%)",
-    3: "hsl(218, 44%, 92%)",
-    2: "hsl(220, 50%, 95%)",
-    1: "hsl(220, 55%, 98%)",
+    17: "hsl(0, 0%, 1%)",
```

Comment:
Unrelated to the actual PR, but is there a reason why we have so many color stops? I think the most I've ever seen is 10–12 (usually mapping to the font weight system of 100-900, with maybe a 50 and a 950 thrown in)

Reply:
I agree this is debt. We should reduce these in the future.